demo.py

```diff
@@ -5,9 +5,9 @@
         'title': 'MyModel',
         'type': 'model',
         'fields': {
-            'name': {'type': 'str'},
-            'age': {'type': 'int', 'ge': 18},
-            'is_developer': {'type': 'bool', 'default': True},
+            'name': {'schema': {'type': 'str'}},
```

Comment:
I think we can probably remove this file, it isn't doing much and will just be out of date as it's not tested.